bindings/rust/evmc-vm/src/container.rs

```diff
@@ -132,7 +132,7 @@ mod tests {
             get_block_hash: None,
             emit_log: None,
         };
-        let mut host_context = ::evmc_sys::evmc_host_context { _unused: [] };
+        let mut host_context = ::evmc_sys::evmc_host_context::default();
```

Comment:
Is it possible to pass a null pointer instead of creating host_context object?

Reply:
Yeah, that should be an option too.

Reply:
Just tried it and it feels like to be simple leaving it as is, for now.

Reply:
I tried it too, but feels like null pointers is Rust's advanced feature.